Makefile

```diff
@@ -27,5 +27,14 @@ eunit:
 qc:
 	$(REBAR) qc
 
+AFL_FUZZ = afl-fuzz
+AFL_CC = afl-gcc
+AFL_CXX = afl-g++
+FUZZ_SKELETON = fuzz_skeleton
```

Comment:
Should use `?=` for these so I can do `FUZZ_SKELETON=../niffy/fuzz_skeleton make fuzz` et cetera.